> From the pipeline result:
> 
>     * _"##[error] Failed"_
> 
> 
> I think the pipeline needs to incorporate more verbose error messages. The faster, the better.

You can see the actual logs by clicking here
![image](https://github.com/user-attachments/assets/ed5ec700-27e0-46d1-99ed-9a9e0d4f1894)
https://dev.azure.com/shine-oss/winget-pkgs/_build/results?buildId=140816&view=artifacts&pathAsName=false&type=publishedArtifacts

--- a/manifests/w/WikimediaProject/AutoWikiBrowser/6.4.0.0/WikimediaProject.AutoWikiBrowser.installer.yaml
+++ b/manifests/w/WikimediaProject/AutoWikiBrowser/6.4.0.0/WikimediaProject.AutoWikiBrowser.installer.yaml
@@ -10,10 +10,10 @@ NestedInstallerFiles:
   PortableCommandAlias: autowikibrowser
 ArchiveBinariesDependOnPath: true
 Dependencies:
-  WindowsFeatures:
-  - NetFx4-AdvSrvs
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
 Installers:
-- Architecture: x86
+- Architecture: x64
   InstallerUrl: https://deac-riga.dl.sourceforge.net/project/autowikibrowser/autowikibrowser/AutoWikiBrowser6400/AutoWikiBrowser6400.zip
   InstallerSha256: 90B91D17509BBDDD8B440FE8F9A6ADB2105862949C0FA3A15A1E9F064C416844
 ManifestType: installer

